### ISSUE TITLE
How to share embeddings between an EmbeddingCollection and an EmbeddingBagCollection?

### ISSUE THREAD

**Issue by tiankongdeguiji** [meta-pytorch/torchrec#2268]
We aim to share embeddings between sparse features and sequence sparse features. For sparse features, we use `EmbeddingBagCollection`, and for sequence sparse features, we use `EmbeddingCollection`. Could you advise on how to effectively share embeddings between an `EmbeddingCollection` and an `EmbeddingBagCollection`?

**Comment by tiankongdeguiji** [comment]
Hi, @henrylhtsang  @IvanKobzarev  @joshuadeng  @PaulZhang12  can you see this problem?

**Comment by PaulZhang12** [comment]
I don't believe this is something that is supported, feel free to create a prototype

**Comment by tiankongdeguiji** [comment]
> I don't believe this is something that is supported, feel free to create a prototype

Thank you! I find this challenging because the states of ebc and ec are created and managed by fbgemm. Do you have any suggestions?
